pkgs/development/libraries/libunity/default.nix

```diff
@@ -32,17 +38,20 @@ stdenv.mkDerivation rec {
     gtk3
   ];
 
-  propagatedBuildInputs = [ dee libdbusmenu ];
+  propagatedBuildInputs = [
+    dee
+    libdbusmenu
+  ];
 
-  preConfigure = "NOCONFIGURE=1 ./autogen.sh";
+  preConfigure = ''
+    intltoolize
```

Comment:
Why use this? The source code still seems to use `gnome-common`.

Reply:
Yeah it does. But It doesn't mean I can't do everything to resist that fact :smile:

Reply:
:rofl: